doc/api.md

```diff
@@ -215,6 +217,27 @@ Note that, when information coming from devices, this means measures, are not de
 device, the IoT agent will store that information into the destination entity using the same attribute name than the
 measure name, unless `explicitAttrs` is defined. Measures `id` or `type` names are invalid, and will be ignored.
 
+## Device autoprovision and entity creation
+
+For those agents that uses IoTA Node LIB version 3.4.0 or higher, you should consider that the entity is not created
+automaticaly when a device is created. This means that al entities into the context broker are created when data 
+arrives from a device, either if the device is created or autoprovisioned.
```

Comment:
```suggestion
automaticaly when a device is created. This means that all entities into the context broker are created when data 
arrives from a device, either if the device is explicitly created (via provisioning API) or autoprovisioned.
```

It could be even better in the "via provisioning API" could be used in a link pointing to the specific section in API Routes with the request to be used in this case.

Reply:
Included here: https://github.com/telefonicaid/iotagent-node-lib/pull/1551/commits/e44e3c3e60807c26309c4c3ef32070175f1bd40e